### DISCUSSION TITLE
How should I open the properties window?

### DISCUSSION THREAD

**Discussion by ilyfairy** [valinet/ExplorerPatcher#2038] [Q&A]
I disabled taskbar changes in advanced settings, and then I can't open properties.

**Comment by Poopooracoocoo** [comment]
https://github.com/valinet/ExplorerPatcher/issues/1962#issuecomment-1699024030

**Comment by pyrates999** [comment]
to run the explorer patcher properties application, run this from the run menu:
C:\Windows\System32\rundll32.exe "C:\Program Files\ExplorerPatcher\ExplorerPatcher.amd64.dll",ZZGUI

Then go to advanced and make sure the option **Do not change the taskbar context menu (e.g. do not display the "Properties"** is not enabled. Then you can right click on the taskbar and you should see the **Properties** option.
